meta-mender-tn-imx-bsp/README.md

```diff
@@ -0,0 +1,41 @@
+# meta-mender-tn-imx-bsp
+
+Mender integration layer for TechNexion family of boards.
+
+The supported and tested boards are:
+
+- [PICO-PI-IMX8M-MINI](https://hub.mender.io/t/technexion-pico-pi-imx8m-mini/2689)
+
+Visit the individual board links above for more information on status of the
+integration and more detailed instructions on how to build and use images
+together with Mender for the mentioned boards.
+
+## Dependencies
+
+This layer depends on [TechNexion Yocto 3.0 Zeus 5.4.y GA BSP](https://github.com/TechNexion/tn-imx-yocto-manifest/tree/zeus_5.4.y-next)
+
+
+## Quick start
+
+The following commands will setup the environment and allow you to build images
+that have Mender integrated.
+
+
+```
+mkdir mender-tn-imx-bsp && cd mender-tn-imx-bsp
+repo init -u https://github.com/TechNexion/tn-imx-yocto-manifest.git -b zeus_5.4.y-next -m imx-5.4.24-2.1.0.xml
+wget --directory-prefix .repo/manifests https://raw.githubusercontent.com/mendersoftware/meta-mender-community/zeus/meta-mender-tn-imx-bsp/scripts/imx-5.4.24-2.1.0_demo_mender.xml
```

Comment:
Should this go into .repo/local_manifests instead?  That's a standard repo feature.

Reply:
Do you know what to specify [this](https://github.com/mendersoftware/meta-mender-community/blob/7bffa1164bc79e77a313b5b4a00f4817f3078e72/meta-mender-tn-imx-bsp/scripts/imx-5.4.24-2.1.0_demo_mender.xml#L4) in that case?

Reply:
Or can that be omitted in that case?

Reply:
Indeed :), updated.